# Analyses of phylogenetics, natural selection, and protein structure of clade 2.3.4.4b H5N1 Influenza A reveal that recent viral lineages have evolved promiscuity in host range and improved replication in mammals in North America

<h3 align="right">Daniel Janies, Kary Ocaña, Sayal Guirales-Medrano, Khaled Obied, Rachel Alexander, and Colby T. Ford</h3>

[![Preprint](https://img.shields.io/badge/bioRxiv-10.1101/2025.03.15.641219-bb2635?style=for-the-badge&logo=read.cv)](https://www.biorxiv.org/content/10.1101/2025.03.15.641219)


## Abstract

H5N1 influenza has been circulating in birds from Eurasia and Africa for more than 146 years, but human infection has been sporadic. H5N1 (clade 2.3.4.4b) has recently infected hundreds of species of wild and domestic birds and mammals in North America. Furthermore, as of February 26, 2025, H5N1 has infected 70 humans in the United States, and one infection proved lethal. Furthermore, in attempts to control H5N1 in the United States, 10s of millions of egg-laying chickens have died or been culled. These efforts have led to very high egg prices in the United States. We have developed an analytical bioinformatics and genomics workflow to understand better how H5N1 is circulating in North America and adapting to new host species. Our workflow consists of: 1) Phylogenetic analyses of large viral sequence datasets to identify subclades of viral lineages causing the current outbreaks in humans and farm animals and closely related viral background lineages. 2) Next, we transfer sequence data from subclades of interest with farm animal and human infection, background data, and vaccine candidate data to analyses of natural selection. 3) Once we identify mutations of interest that underlie recent viral adaptation to animal and human infection, we perform computational structural analyses of binding to host proteins for cell receptors and immune processes. Here, we show that H5N1 (clade 2.3.4.4b) is spreading in North America as two distinct subclades of interest for human and animal health. These viral lineages have achieved a vast host range by efficiently binding the viral surface protein Hemagglutinin (HA) to both mammalian and avian receptors. This novel promiscuity of host range is concomitant with the additional strengthening of the polymerase basic 2 (PB2) viral protein’s binding for mammalian and avian immune proteins. Once bound, the immune proteins are disabled, thus allowing for more efficient replication of H5N1 in mammalian and avian cells than seen in the recent past. In conclusion, H5N1 (clade 2.3.4.4b) is causing an animal pandemic through promiscuity of host rage and strengthening ability to evade the innate immune systems of both mammalian and avian cells.
